package.json

```diff
@@ -50,18 +50,19 @@
     "fast-memoize": "2.x.x",
     "immer": "3.x.x",
     "lodash": "4.x.x",
-    "urijs": "1.x.x"
+    "urijs": "1.x.x",
+    "vscode-uri": "2.x.x"
   },
   "devDependencies": {
-    "@stoplight/scripts": "7.0.0",
-    "@types/jest": "^24.0.13",
-    "@types/lodash": "4.14.133",
+    "@stoplight/scripts": "5.x.x",
```

Comment:
bahaha you had to _downgrade_ `@stoplight/scripts`? Uh oh!

Reply:
Yeah, Rollup is a bit of an overkill for cases like this one.